aiven/client/cli.py

```diff
@@ -5537,6 +5535,32 @@ def service__flink__cancel_application_deployments(self) -> None:
             ),
         )
 
+    @arg.json
+    @arg("-n", "--name", required=True, help="Name of the organization to create")
```

Comment:
I would put this as an argument rather than an option (especially since it's `required=True`) ... it does feel like we have a bit of everything in this respect, but `service_name` is an argument as well.